governance/pkg/api/accessrule.go

```diff
@@ -63,6 +63,38 @@ func (a *API) GovCreateAccessRule(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//check if user and group exists
+
+	for _, u := range *createRequest.Approval.Users {
```

Comment:
@meyerjrr could you please shift this into the `rulesvc` package so that it remains consistent between the user and governance API?

Additionally can you please add new test cases that add coverage over this?

Also, the current behaviour will make one query to DynamoDB at a time for each user/group that is an approver. You can use an [`errgroup`](https://pkg.go.dev/golang.org/x/sync/errgroup) to run these checks concurrently. @JoshuaWilkes can help if you are unsure